sdk/purview/purview-datamap-rest/src/serializeHelper.ts

```diff
@@ -1,13 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-export function buildMultiCollection(items: string[], parameterName: string): string {
```

Comment:
this is expected and relevant parameter change is already applied in refreshing pr https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/purview/purview-datamap-rest/src/parameters.ts#L58. not sure why this helper is not removed.